README.md

```diff
@@ -432,6 +432,7 @@ Variable | Description | Default value
 `JIGASI_TRANSCRIBER_ADVERTISE_URL` | Jigasi will post an URL to the chat with transcription file | true
 `DISABLE_HTTPS` | Handle TLS connections outside of this setup | 1
 `ENABLE_HTTP_REDIRECT` | Redirect HTTP traffic to HTTPS (necessary for Let's Encrypt) | 1
+`STUN_SERVERS` | STUN servers used by web clients for P2P connections | meet-jit-si-turnrelay.jitsi.net:443
```

Comment:
Wouldn't it make more sense to merge this with `JVB_STUN_SERVERS`?

Reply:
Indeed, I thought about that, too. However, as already stated in the initial comment, I was unsure if it would make sense to keep these two options separate for some setups and wanted to get some feedback on that.

Reply:
This change need to be applied to https://github.com/jitsi/handbook/blob/master/docs/devops-guide/docker.md (cf: #601)